utils/configuration.go

```diff
@@ -244,3 +246,20 @@ func AdjustLogLevel(increment bool) error {
 	logrus.SetLevel(lvl)
 	return nil
 }
+
+// SetupSignalTrap is a utility to trap supported signals hand handle them (currently by increasing logging)
+func SetupSignalTrap(handler func(os.Signal)) chan os.Signal {
+	if len(notary.NotarySupportedSignals) == 0 {
```

Comment:
non-blocking-nit: can we test this case with some mocking?  I'm fine with assigning this const to empty for the test and deferring to return it back to its original value, to mock what Windows would do